allennlp/modules/token_embedders/pretrained_transformer_mismatched_embedder.py

```diff
@@ -0,0 +1,65 @@
+from overrides import overrides
+import torch
+
+from allennlp.modules.token_embedders import PretrainedTransformerEmbedder, TokenEmbedder
+from allennlp.nn import util
+
+
+@TokenEmbedder.register("pretrained_transformer_mismatched")
+class PretrainedTransformerMismatchedEmbedder(TokenEmbedder):
+    """
+    Use this embedder to embed wordpieces given by `PretrainedTransformerMismatchedIndexer`
+    and to pool the resulting vectors to get word-level representations.
+    """
+
+    def __init__(self, model_name: str) -> None:
```

Comment:
Could you add a docstring to this class?

Reply:
You mean the `# Parameters` section?

Reply:
Yes - someone new to the library will not necessarily know what `model_name` means. It also helps the classes render nicely in the docs.

Reply:
Done